src/IdentityDecorator.php

```diff
@@ -76,9 +76,9 @@ public function canResult(string $action, $resource): ResultInterface
     /**
      * @inheritDoc
      */
-    public function applyScope(string $action, $resource): mixed
+    public function applyScope(string $action, $resource, mixed ...$optionalArgs): mixed
```

Comment:
```suggestion
    public function applyScope(string $action, mixed $resource, mixed ...$optionalArgs): mixed
```
Seems the type for `$resource` was missed earlier.